javascript/javascript3/week3/art/main.js

```diff
@@ -0,0 +1,80 @@
+//Obtaining HTML elements
+const canvas = document.querySelector("canvas");
+
+//Making canvas fullscreen
+canvas.height = screen.availHeight;
+canvas.width = screen.availWidth;
+
+//Make some art using classes
+//Task 1 - Paint a circle to a canvas element
+const ctx = canvas.getContext("2d");
+ctx.beginPath();
+ctx.arc(100, 75, 50, 0, 2 * Math.PI);
+ctx.fillStyle = "#00BFFF";
+ctx.fill();
+ctx.strokeStyle = "white";
+ctx.stroke();
+
+// Task 2 - Circle class
+class Circle {
+  constructor(x, y, r, startAngle, endAngle, fillColor) {
+    this.x = x;
+    this.y = y;
+    this.r = r;
+    this.startAngle = startAngle;
+    this.endAngle = endAngle;
+    this.fillColor = fillColor;
+  }
+
+  draw() {
+    ctx.beginPath();
+    ctx.arc(this.x, this.y, this.r, this.startAngle, this.endAngle);
+    ctx.fillStyle = this.fillColor;
+    ctx.fill();
+    ctx.stroke();
+  }
+}
```

Comment:
To make the class reusable, we can pass "context" as an argument to the draw method. 
  draw(context) {
      context.beginPath();
      context.arc(this.x, this.y, this.r, this.startAngle, this.endAngle);
      context.fillStyle = this.fillColor;
      context.fill();
      context.stroke();
  }

const c1 = new Circle(50, 50, 20, 0, 2 * Math.PI, "#000000");
c1.draw(ctx);

Reply:
Fixed! Thank you for the feedback and really cool hints :)